USFMConverter/Core/Util/SettingManager.cs

```diff
@@ -16,10 +16,10 @@ public static class SettingManager
 
         static SettingManager()
         {
-            string localAppData = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
+            string userDir = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
```

Comment:
The docs warn against using UserProfile and recommend ApplicationData instead.  Have you tried using ApplicationData instead of LocalApplicationData?

https://docs.microsoft.com/en-us/dotnet/api/system.environment.specialfolder?view=net-5.0#System_Environment_SpecialFolder_UserProfile

https://docs.microsoft.com/en-us/dotnet/api/system.environment.specialfolder?view=net-5.0#System_Environment_SpecialFolder_ApplicationData